package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-plugin-shopify",
-  "version": "16.0.1",
+  "version": "17.0.0-1",
```

Comment:
Will this be reverted back to `16.0.1` before merging?

I believe we leave this untouched. When we 🚢 a release, `yarn version --new-version <major|minor|patch>` will bump it up for us.